### PR TITLE
Docs: Added :class:`~.SurroundingRectangle` to example gallery

### PR DESCRIPTION
**Pull request by kolibril13**
## Motivation:

This pr will make some further improvements in the docs (small tweaking of the example page)
see example here: https://manimce--1036.org.readthedocs.build/en/1036/examples.html#movinggrouptodestiny
EDIT: As this is still work in progress, I marked it as a Draft.
## Acknowledgements
- [ x ] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)

## Reviewer Checklist
- [ ] Newly added functions/classes are either private or have a docstring
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings
- [ ] The oneline summary has been included [in the wiki](https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release)

### CODE REVIEW

File: docs/source/examples.rst [hunk 1]
```diff
@@ -94,7 +94,8 @@ Basic Concepts
                 [[i * 256 / n for i in range(0, n)] for _ in range(0, n)]
             )
             image = ImageMobject(imageArray).scale(2)
-            self.add(image)
+            image.background_rectangle= SurroundingRectangle(image, GREEN)
```
Comment (jsonvillanueva) [inline docs/source/examples.rst:97]:
```suggestion
            image.background_rectangle = SurroundingRectangle(image, GREEN)
```

**Review by jsonvillanueva — APPROVED** [review]

**Comment by behackl** [comment]
@kolibril13, what is the status of this PR? Are you still working on it? Should it be merged?

**Comment by kolibril13** [comment]
Yess, it can get merged.

File: docs/source/examples.rst [hunk 2]
```diff
@@ -213,6 +214,16 @@ Animations
             self.play(square.animate.scale(0.3))
             self.play(square.animate.rotate(0.4))
 
+.. manim:: MovingGroupToDestiny
+
+    class MovingGroupToDestiny(Scene):
```
Comment (behackl) [inline docs/source/examples.rst:219]:
```suggestion
.. manim:: MovingGroupToDestination

    class MovingGroupToDestination(Scene):
```

**Review by behackl — CHANGES_REQUESTED** [review]
LGTM, apart from the scene name of the new example.

**Review by behackl — APPROVED** [review]

**Pull request merged**